packages/elements-core/src/components/Docs/HttpOperation/HttpOperation.tsx

```diff
@@ -34,28 +36,28 @@ const HttpOperationComponent = React.memo<HttpOperationProps>(
     const prettyName = (data.summary || data.iid || '').trim();
     const hasBadges = isDeprecated || isInternal;
 
-    const header = (!layoutOptions?.noHeading || hasBadges) && (
-      <VStack spacing={5}>
-        <HStack spacing={5}>
-          {!layoutOptions?.noHeading && prettyName ? (
-            <Heading size={1} fontWeight="semibold">
-              {prettyName}
-            </Heading>
-          ) : null}
-
-          <HStack spacing={2}>
-            {isDeprecated && <DeprecatedBadge />}
-            {isInternal && <InternalBadge isHttpService />}
-          </HStack>
-        </HStack>
-
-        <MethodPath method={data.method} path={data.path} />
-      </VStack>
+    const header = (
+      <OperationHeader
+        id={data.id}
+        method={data.method}
+        path={data.path}
+        noHeading={layoutOptions?.noHeading}
+        hasBadges={hasBadges}
+        name={prettyName}
+        isDeprecated={isDeprecated}
+        isInternal={isInternal}
+      />
     );
 
+    const descriptionChanged = nodeHasChanged?.({ nodeId: data.id, attr: 'description' });
```

Comment:
These `nodeHasChanged` calls are effectively noops for all existing users of elements (since it's expected that platform is really the only consumer of elements that leverages the new `nodeHasChanged` prop).